include/nbl/video/IGPUCommandBuffer.h

```diff
@@ -97,7 +97,7 @@ class NBL_API IGPUCommandBuffer :
             assert(false);
             return false;
         }
-        return base_t::reset(_flags);
+        return resetCommon();
```

Comment:
check for commandpool reset first, then check valid usage, see my other comment

also call it `reset_impl` not `resetCommon`

Reply:
furthermore, if the commandbuffer is not indiviually resettable it should go into INVALID instead of INITIAL (see my other comment)

Reply:
Resolved.